.github/workflows/cezzis-api-cicd.yaml

```
@@ -21,8 +21,31 @@
   workflow_dispatch:
 
 jobs:
+  gitVersion:
+    name: Git version
+    runs-on: ubuntu-latest
+    outputs:
+      semVer: ${{ steps.gitVersion.outputs.semVer }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)